File: src/Server/DTO/ClientMessageDTO.java
package Server.DTO;

/**
    The ClientMessageDTO class represents the client message as a 
    Data Transfer Object.
 */

public class ClientMessageDTO{
    private String plainText;
    private String userName;
  /**
    Creates a new instance of the class ClientMessageDTO.
    @param msg String containing message from user.
    @param user String with the user name.
   */
    public ClientMessageDTO(String msg, String user){
        this.plainText = msg;
        this.userName = user;
    }

    public String getPlainText(){
        return this.plainText;
    }
    
    public String getUserName(){
        return this.userName;
    }

    public void setPlainText(String msg){
        this.plainText = msg;
    }

    public void setUserName(String user){
        this.userName = user;
    }
}

File: homepage/src/backend/DTO/ClientMessageDTO.js
/**
 * The ClientMessageDTO class represents the client message as a
 * Data Transfer Object.
 */

export class ClientMessageDTO {
    plainText;
    userName;

    /**
     * Creates a new instance of the class ClientMessageDTO.
     * 
     * @param msg  String containing message from user.
     * @param user String with the user name.
     */
    ClientMessageDTO(msg, user) {
        this.plainText = msg;
        this.userName = user;
    }

    getPlainText() {
        return this.plainText;
    }

    getUserName() {
        return this.userName;
    }

    setPlainText(msg) {
        this.plainText = msg;
    }

    setUserName(user) {
        this.userName = user;
    }
}